### PR TITLE
Merge pull request #30 from ai-cfia/28-louis-db-readme-improvment

### PR DESCRIPTION
**Pull request by leejaeka**
Adding 'vis' folder to contain vis.ipynb notebook that outputs 2d visualisation of vector space

### CODE REVIEW

File: vis/notebook/vis.ipynb [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# Run this if you don't have a nomic access token\n",
+        "#!nomic login\n",
+        "# Run this if you have nomic access token\n",
+        "!nomic login 'HfFeBu516-CFtRnOFlEY9k-A5DWMqyKrhNe7XyxSv3irV'\n",
```
Comment (rngadam) [inline vis/notebook/vis.ipynb:55]: note that you've embedded the nomic login token into the notebook source; it's bad practice to store secrets in the source.

there's a way to use secrets from codespaces: https://docs.github.com/en/codespaces/managing-your-codespaces/managing-secrets-for-your-codespaces

these show up as environment variables.

**Review by rngadam — CHANGES_REQUESTED** [review]
you should create a corresponding issue and create a branch based on that issue name. see Editing-code-in-WSL-with-Visual-Studio-Code in the internal wiki

**Review by leejaeka — COMMENTED** [review]
replaced nomic api key with github codespace secrets

File: vis/notebook/vis.ipynb [hunk 2]
```diff
@@ -49,10 +49,12 @@
       "metadata": {},
       "outputs": [],
       "source": [
+        "import os\n",
+        "NOMIC_TONY_API_TOKEN = os.environ['NOMIC_TONY_API_TOKEN']\n",
         "# Run this if you don't have a nomic access token\n",
         "#!nomic login\n",
         "# Run this if you have nomic access token\n",
-        "!nomic login 'HfFeBu516-CFtRnOFlEY9k-A5DWMqyKrhNe7XyxSv3irV'\n",
```
Comment (rngadam) [inline vis/notebook/vis.ipynb]: the key is still in the history.

1) rebase to squash your changes into a single commit.
2) make sure you've invalidated this key so it doesn't work anymore (as you have to assume someone scraped it)

**Review by rngadam — CHANGES_REQUESTED** [review]

**Pull request closed without merging**